sonic-psud/scripts/psud

```diff
@@ -125,15 +125,6 @@ def get_psu_key(psu_index):
     return PSU_INFO_KEY_TEMPLATE.format(psu_index)
 
 
-def psu_db_update(psu_tbl, psu_num):
```

Comment:
since you have deprecated this function, make sure no unit test will be broken.

Reply:
Thanks, I have deprecated the single test case dedicated to this function and adjust others that related to this change.